software/gatk4/variantfiltration/main.nf

```diff
@@ -0,0 +1,42 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process GATK4_VARIANTFILTRATION {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.2.0.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/gatk4:4.2.0.0--0"
+    } else {
+        container "quay.io/biocontainers/gatk4:4.2.0.0--0"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+    tuple path(fasta), path(fai), path(dict)
+
+    output:
+    tuple val(meta), path("*filtered.vcf"), emit: vcf
+    path "*.version.txt"          , emit: version
```

Comment:
```suggestion
    tuple val(meta), path("*filtered.vcf"), emit: vcf
    path "*.version.txt"                  , emit: version
```

Reply:
Seriously! Is it only me that sees this 😆

Reply:
👀